pygmt/src/basemap.py

```diff
@@ -56,7 +56,8 @@ def basemap(self, **kwargs):
     {R}
     {B}
     map_scale : str
-        ``'[g|j|J|n|x]refpoint'``
+        [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*
+        +\ **w**\ *length*.
         Draws a simple map scale centered on the reference point specified.
```

Comment:
Need a `\` at the end of the first line. Otherwise, there will be a whitespace after *refpoint*
```suggestion
        [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*\ 
        **+w**\ *length*.
        Draws a simple map scale centered on the reference point specified.
```